apps/experiments/forms.py

```diff
@@ -291,24 +293,44 @@ class TranslateMessagesForm(forms.Form):
         label="Select Language",
         widget=forms.Select(attrs={"class": "select select-bordered w-full", "id": "translation-language"}),
     )
-    provider_model = forms.ChoiceField(
+    llm_provider = forms.ChoiceField(
         choices=[],
         required=True,
-        label="Select LLM Provider Model",
+        label="Select LLM Provider",
+        widget=forms.Select(attrs={"class": "select select-bordered w-full", "id": "translation-provider"}),
+    )
+    llm_provider_model = forms.ChoiceField(
+        choices=[],
+        required=True,
+        label="Select LLM Model",
         widget=forms.Select(attrs={"class": "select select-bordered w-full", "id": "translation-provider-model"}),
     )
 
     def __init__(self, *args, team, translatable_languages, is_translate_all_form=False, **kwargs):
         super().__init__(*args, **kwargs)
 
-        self.fields["provider_model"].choices = [
-            ("", "Choose a model for translation")
-        ] + get_dropdown_llm_model_choices(team)
+        providers = get_llm_provider_by_team(team)
+        provider_choices = list(providers.values_list("id", "type"))
```

Comment:
It will help if we can show the provider name instead of the type (or the type can be in brackets or something). This would make it easier to identify the providers, especially if I have more than one provider of the same type.

I.e.
<img width="510" height="472" alt="image" src="https://github.com/user-attachments/assets/b74c35a9-2952-45b9-9f55-3b741319e72f" />